config.json

```diff
@@ -458,24 +471,34 @@
         "slug": "raindrops",
         "name": "Raindrops",
         "uuid": "e3e73171-c3d1-49f7-9511-d1f4a08e7a69",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "strings"
+        ],
+        "prerequisites": [
+          "strings"
+        ],
         "difficulty": 2,
         "topics": [
           "filtering",
-          "text_formatting"
+          "text_formatting",
+          "strings"
         ]
       },
       {
         "slug": "robot-simulator",
         "name": "Robot Simulator",
         "uuid": "6b8f952e-94ea-4264-accf-cdb28b3b4529",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "strings"
+        ],
+        "prerequisites": [
+          "strings"
+        ],
         "difficulty": 3,
         "topics": [
           "records",
-          "tuples"
+          "tuples",
+          "strings"
```

Comment:
The topics are deprecated. They don't do anything but can still be useful to fill the other fields, so we keep them around, but could you remove all "strings" from all of the topics? Now that we have reviewed the concept, it won't be useful to keep it there.

Reply:
Done